modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ImportAndDeleteObjectEntryPerformanceTest.java

```diff
@@ -76,8 +77,14 @@ private String _createObjectEntryJSON() throws Exception {
 	}
 
 	private void _deleteObjectEntry() throws Exception {
-		try (LoggingTimer loggingTimer = new LoggingTimer(
-				"Delete " + _OBJECT_ENTRY_COUNT + " Object Entries")) {
+		StringBundler sb = new StringBundler(3);
+
+		sb.append(" Delete ");
+		sb.append(_OBJECT_ENTRY_COUNT);
+		sb.append(" Object Entries");
+
+		try (PerformanceTimer performanceTimer = new PerformanceTimer(
+				5000, sb)) {
```

Comment:
@slnn Plz make the parameter a `String`, and pass `StringBundler.concat` or just use `+` as SF prompts. Also, if the `String` parameter is called `message` or `name` or `description`, put it in front of the timeout.

Reply:
Done